frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -326,29 +327,14 @@ function FileManagerMenu:setUpdateItemTable()
                     {
                         text = _("Set home folder"),
                         callback = function()
-                            local text
-                            local home_dir = G_reader_settings:readSetting("home_dir")
-                            if home_dir then
-                                text = T(_("Home folder is set to:\n%1"), home_dir)
-                            else
-                                text = _("Home folder is not set.")
-                                home_dir = Device.home_dir
+                            local title_header = _("Current home folder:")
```

Comment:
(Not unusual your aggresive deduplication :) We're losing a bit of the verbosity, which was a nice touch and fine with these rarely used features. But ok, as it's rarely used, I don't really mind.)

What I feel a bit odd is the splitting of the "title" (the multilines string above the buttons") among 2 widget/function: this here with the `:` at the end, making it a bit gramatically incomplete in the context here, and some `\n`+path added in the function elsewhere.
May be a `\n%1` in here would feel better? With an uncondtional T(title_header, path) on the other side?

Reply:
This allows to fix the next comment with a change to one file only.

Reply:
Still :) I then could argue why the `:` is included in the first part, and why wouldn't you append `:\n\n%1\n` in the other function :) (not asking you to do that! I still prefer this with the `:` in the first part).
I still feel it's more natural for translators to have to translate `Current home folder:\n\n%1\n` than just `Current home folder:`. (But I don't do any translation, so what do I know? :))

Reply:
I think that `:` shows the context and may affect and help to translators
`\n` adds nothing to understanding and just makes duplications.
I do not translate anything either so asking @Frenzie.